src/main/java/com/exasol/releaserobot/repository/maven/MavenPom.java

```diff
@@ -33,12 +47,43 @@ public String getVersion() {
     }
 
     /**
-     * Get deliverable name.
+     * Check if version presents.
+     *
+     * @return true if version presents
+     */
+    public boolean hasVersion() {
```

Comment:
I see that the `getVersion` method is used in several places, but it looks like this value might not be always here. Could be any problem?

Reply:
Added hasVersion checks in the place I used getVersion